flight/Modules/ManualControl/tablet_control.c

```diff
@@ -97,8 +97,9 @@ int32_t tablet_control_select(bool reset_controller)
 
 			// Command to not move. This code is identical to that in manualcontrol
 			LoiterCommandData loiterCommand;
-			loiterCommand.Forward = 0;
-			loiterCommand.Right = 0;
+			loiterCommand.Pitch = 0;
+			loiterCommand.Roll = 0;
+			loiterCommand.Throttle = 0.5f;
```

Comment:
Right now throttle is ignored.  I have a pending issue to enable user control of altitude, in which case 0.5 will be inside the deadband which means "don't climb or descend".